src/github.ts

```diff
@@ -13,15 +13,15 @@ export function getOctokitSingleton() {
   return octokitSingleton;
 }
 
-export async function listTags() {
+export async function listTags(): Promise<any> {
```

Comment:
I'd rather return a proper type here rather than changing it everywhere

Reply:
I have no understanding of TypeScript but I did dive thru the associated type lib.

`paginate` doesn't appear to expose a typedef or it returns an Any.

Any advice would be helpful.

Reply:
Hello, what is the condition of this pull request? Any Chance that it will get merged? Or do you want to address the "pagination" issue somewhere else?

Reply:
I have to fix the TS definitions before merging it. Haven't had the time to do so yet